packages/tools/devtools/devtools-core/src/data-visualization/DefaultVisualizers.ts

```diff
@@ -260,26 +265,33 @@ export const visualizeSharedTree: VisualizeSharedObject = async (
 
 	// Root node of the SharedTree's content.
 	const treeView = sharedTree.exportVerbose();
-	// TODO: this visualizer doesn't consider the root as a field, and thus does not display the allowed types or handle when it is empty.
-	// Tracked by https://dev.azure.com/fluidframework/internal/_workitems/edit/26472.
 	if (treeView === undefined) {
 		throw new Error("Support for visualizing empty trees is not implemented");
 	}
 
 	// Schema of the tree node.
 	const treeSchema = sharedTree.exportSimpleSchema();
 
-	// Traverses the SharedTree and generates a visual representation of the tree and its schema.
-	const visualTreeRepresentation = await visualizeSharedTreeNodeBySchema(
-		treeView,
-		treeSchema,
-		visualizeChildData,
-	);
+	const sf = new SchemaFactory(undefined);
+	const schemaName = Tree.is(treeView, [sf.boolean, sf.null, sf.number, sf.handle, sf.string])
```

Comment:
Do we need to special-case primitives here? Can we not just use `Tree.schema(treeView).identifier`

Reply:
[TreeNodeApi.schema](https://github.com/microsoft/FluidFramework/blob/e533d64f639897154eeabbdde72f891b4ceed142/packages/dds/tree/src/simple-tree/api/treeNodeApi.ts#L60) accepts either a `TreeNode` or a `TreeLeafValue`. 

If `TreeView` is a non-leaf node (thus has a type of `VerboseTree`), I believe we should use `type` to get its metadata.

Reply:
But that case should correspond to the `TreeNode` case, right?